Tests/SonarQube.TeamBuild.PreProcessor.Tests/TargetsInstallerTests.cs

```diff
@@ -115,15 +121,16 @@ private static void InstallTargetsFileAndAssert(string expectedContent, bool exp
                     ".targets does not have expected content at " + path);
 
                 Assert.IsTrue(logger.DebugMessages.Any(m => m.Contains(destinationDir)));
+            }
 
-                if (expectCopy)
-                {
-                    Assert.AreEqual(
-                        TargetsInstaller.DestinationDirs.Count, 
-                        logger.DebugMessages.Count,
-                        "All destinations should have been covered");
-                }
+            if (expectCopy)
```

Comment:
I moved this block of code outside the loop as it doesn't use any of the loop variables.